### ISSUE TITLE
`dc9166f1253df` Alert: "Setting adjustable number of callback queues."

### ISSUE THREAD

**Issue by gctucker** [kernelci/kernelci-project#78]
* Commit: `dc9166f1253df rcu-tasks: Use more callback queues if contention encountered`
* Regressions:
  * https://linux.kernelci.org/test/case/id/61a4f02ad5a9e3a5f218f6f6/
* Email: https://lore.kernel.org/lkml/42cba265-2616-9382-324f-11718c8dca0c@collabora.com/
* Bisection report:
```
next/master bisection: baseline.dmesg.alert on qemu_arm64-virt-gicv3

Summary:
  Start:      f30a24ed97b40 Add linux-next specific files for 20211126
  Plain log:  https://storage.kernelci.org/next/master/next-20211126/arm64/defconfig+CONFIG_ARM64_64K_PAGES=y/clang-13/lab-collabora/baseline-qemu_arm64-virt-gicv3.txt
  HTML log:   https://storage.kernelci.org/next/master/next-20211126/arm64/defconfig+CONFIG_ARM64_64K_PAGES=y/clang-13/lab-collabora/baseline-qemu_arm64-virt-gicv3.html
  Result:     dc9166f1253df rcu-tasks: Use more callback queues if contention encountered

Checks:
  revert:     PASS
  verify:     PASS

Parameters:
  Tree:       next
  URL:        https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git
  Branch:     master
  Target:     qemu_arm64-virt-gicv3
  CPU arch:   arm64
  Lab:        lab-collabora
  Compiler:   clang-13
  Config:     defconfig+CONFIG_ARM64_64K_PAGES=y
  Test case:  baseline.dmesg.alert

Breaking commit found:

-------------------------------------------------------------------------------
commit dc9166f1253df20e01b0132ece22efaeb5b0a7e0
Author: Paul E. McKenney <paulmck@kernel.org>
Date:   Wed Nov 24 15:12:15 2021 -0800

    rcu-tasks: Use more callback queues if contention encountered
```

**Comment by gctucker** [comment]
The outcome of the discussion is that the message will be converted to `pr_info()`.

**Comment by gctucker** [comment]
This patch was updated before being applied, then it got merged as `ab97152f88a4d580b89f0b7cc3028ffac438216f` which doesn't show the issue any more.

**Comment by gctucker** [comment]
Verified: https://linux.kernelci.org/test/plan/id/61efbdf289fe9d6e84abbd41/
